cra-config.yaml

```diff
@@ -10,3 +10,4 @@ CRA_TARGETS:
     CRA_ENVIRONMENT_VARIABLES:  # An optional map of environment variables for CRA, where the key is the variable name and value is the value. Useful for providing TF_VARs.
       TF_VAR_kms_key_crn: "crn:v1:bluemix:public:hs-crypto:us-south:a/abac0df06b644a9cabc6e44f55b3880e:e6dce284-e80f-46e1-a3c1-830f7adff7a9:key:1368d2eb-3ed0-4a8b-b09c-2155895f01ea"
       TF_VAR_prefix: "test"
+      TF_VAR_allowed_ip_addresses: "[\"0.0.0.0/0\"]"
```

Comment:
actually, for CRA we will want to use some locked down IP (as this means allow all, so might fail scan some day). Set it to `"[\"17.5.7.8.0/16\"]"` for example - it wont actually apply this for CRA scan, just does a plan